gov/politeia/types/types.go

```diff
@@ -240,12 +240,12 @@ type ProposalMetadata struct {
 // prepare figures for display. Many of these manipulations require a tip height
 // and a target block time for the network, so those must be provided as
 // arguments.
-func (pinfo *ProposalInfo) Metatdata(tip, targetBlockTime int64) *ProposalMetadata {
+func (pinfo *ProposalInfo) Metadata(tip, targetBlockTime int64) *ProposalMetadata {
 	meta := new(ProposalMetadata)
 	desc := pinfo.VoteStatus.ShortDesc()
 	switch desc {
 	case "Authorized":
-		blocksTil := windowSize - tip%windowSize
+		blocksTil := int64(windowSize)
```

Comment:
You're correct that this was wrong, but I'm not sure this fixes it. Actually, I'm not sure we can know the `StartHeight` on an `"Authorized"` proposal. It starts whenever the admin approves. That was my mistake.